### PR TITLE
Set bounce_margin_factor default to 0.95

### PR DESCRIPTION
**Pull request by EvanKrall**
In COMPINFRA-2270, several service owners are complaining about stuck deployments because of individual pods that never become healthy.

This change would make it so any service with 20-39 containers would ignore a single stuck pod; 40-59 would ignore 2, etc.

From what I can tell from the [code that manages PodDisruptionBudgets](https://github.com/Yelp/paasta/blob/02b165ac877700b28f98653f8ad2b30f0d73f339/paasta_tools/kubernetes/application/controller_wrappers.py#L147), this won't affect PDBs - anything that doesn't have `bounce_margin_factor` set gets the default PDB from SystemPaastaConfig.

### CODE REVIEW

**Review by nemacysts — APPROVED** [review]

**Comment by nemacysts** [comment]
(might be good to add a description to this PR before merging)

File: tests/test_kubernetes_tools.py
Comment (nemacysts) [inline tests/test_kubernetes_tools.py:4173]: looks like something funky happened here with the merge markers?

File: tests/test_kubernetes_tools.py
Comment (EvanKrall) [inline tests/test_kubernetes_tools.py:4179]: lol

**Review by nemacysts — APPROVED** [review]
(we should hold off on merging this until we finish the karpenter upgrade - so maybe tomorrow?)

**Comment by nemacysts** [comment]
(should be safe to merge now, i think - if we pull master the one test failure should go away too)

**Comment by nemacysts** [comment]
@EvanKrall i've merged master into this to get a green test run - probably good to merge after that imo :)

**Review by bobokun — APPROVED** [review]

**Review by bobokun — APPROVED** [review]

**Review by jfongatyelp — APPROVED** [review]

**Pull request merged**
